pkgs/applications/version-management/cvs/default.nix

```diff
@@ -28,14 +28,21 @@ stdenv.mkDerivation {
     done
   '';
 
-  buildInputs = [ nano ];
+  configureFlags = [
+    "--with-editor=${nano}/bin/nano"
+    "cvs_cv_func_printf_ptr=yes"
```

Comment:
Btw: It might make sense to briefly document why `cvs_cv_func_printf_ptr=yes` is required (e.g. to fix the cross-compilation?) as that isn't obvious to me but I don't really mind either.